### ISSUE TITLE
What can we learn and implement from the Docker strategy of other webmail project?

### ISSUE THREAD

**Issue by marclaporte** [cypht-org/cypht#1146]
### Version & Environment

Rev: []

OS:  []

### Steps to reproduce
1.
2.
3.
4. ...
5. Profit

**Comment by jonocodes** [comment]
Good idea. Here are the things that I think come up here after looking at those repos:

# 1. ssl/https

Snappy shows the use of self signed certs, presumably in development. I tend to think this is not a good idea. But we can discuss it. 
In terms of production deployment, we should definitely tell/recommend that people use ssl. I dont think we should/need to tell them how to do that since there are many ways. That being said we can link to one of our pages that shows one possible way to do it for those who are not familiar. (I thought there was a wiki page here that used caddy or traefik or something.)

# 2. imprecise version numbering

We can point tags to eachother like:
2 -> 2.1 -> 2.1.3

Such that the user can pull just the major version and not worry about the minor version. This is a bit hacky, but fairly common in docker. I tend to think its not worth the trouble, but we can talk more about it.

# 3. configurable logging

We should be using env vars to tune logging, as that is common practice in or outside of docker. There has been some discussion around this here:
https://github.com/cypht-org/cypht/issues/1027

**Comment by marclaporte** [comment]
Related: https://github.com/cypht-org/cypht/issues/1175

**Comment by marclaporte** [comment]
@rodriguezny @kroky @josaphatim @Shadow243 @Baraka24 

Thoughts?
